virtio-devices/src/net.rs

```diff
@@ -400,9 +398,7 @@ impl Net {
             | 1 << VIRTIO_NET_F_HOST_TSO4
             | 1 << VIRTIO_NET_F_HOST_TSO6
             | 1 << VIRTIO_NET_F_HOST_UFO
-            | 1 << VIRTIO_NET_F_MRG_RXBUF
             | 1 << VIRTIO_RING_F_EVENT_IDX
-            | 1 << VIRTIO_RING_F_INDIRECT_DESC
```

Comment:
One small question, have you tried enabling only one of the two virtio features? Might still be interesting to keep one of them if they're not causing any harm.

Reply:
Yes, we should investigate what the underlying cause of the regression is as afterall the patch was an optimisation!